vendor/github.com/opencontainers/runtime-spec/specs-go/config.go

```diff
@@ -320,6 +320,13 @@ type LinuxNetwork struct {
 	Priorities []LinuxInterfacePriority `json:"priorities,omitempty"`
 }
 
+// VTPM definition
+type VTPM struct {
```

Comment:
@crosbymichael How do you feel about including changes to `vendor/`?
@stefanberger Is there a runtime-spec PR open about this?

Reply:
@cyphar I opened https://github.com/opencontainers/runtime-spec/pull/920 for vTPM support

Reply:
What's current status of https://github.com/opencontainers/runtime-spec/pull/920 ?

cc @vbatts